golang/cosmos/upgradegaia.sh

```diff
@@ -0,0 +1,62 @@
+#! /bin/bash
+# 3-way-merge changes from current repo to agoric-sdk/golang/cosmos
+#
+# Usage: ./upgradegaia.sh <FROM_BRANCH> <TO_BRANCH>
+# Such as:
+#  cd ~/src/gaia
+#  /path/to/agoric-sdk/golang/cosmos/upgradegaia.sh release/v6.0.0-rc3 release/v6.0.0
+set -ueo pipefail
+
+FROM_BRANCH="$1"
+TO_BRANCH="$2"
+
+thisdir="$(cd "$(dirname "${BASH_SOURCE[0]}")" > /dev/null && pwd)"
+tmp="$(mktemp -dt upgradegaia.XXXXXX)"
+rm -rf "$tmp"
```

Comment:
I don't believe there's a need to remove the contents - on a successful return from `mktemp`, it's guaranteed to be empty, and leaving the directory prevents a race with a concurrent invocation of this script.